cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -59,6 +59,8 @@
 import java.util.TreeMap;
 import java.util.function.BiFunction;
 
+import static io.strimzi.operator.common.operator.resource.AbstractScalableResourceOperator.ANNOTATION_MANUAL_RESTART;
```

Comment:
The annotation should be probably defined in this file in the first place? Not sure why we import it from `bstractScalableResourceOperator.java` where it doesn't seem to be used at all.